lms/djangoapps/grades/models.py

```diff
@@ -16,6 +16,7 @@
 
 from django.apps import apps
 from django.db import models, IntegrityError, transaction
+from django.contrib.auth import get_user_model
```

Comment:
I think you are not using this import

Reply:
Yes, you are right I have already fixed it.